### PR TITLE
Docs fix

### PR DESCRIPTION
**Pull request by TennisBowling**

### CODE REVIEW

File: docs/the_nimbus_book/src/install.md [hunk 1]
```diff
@@ -6,17 +6,28 @@ The Nimbus beacon chain can run on Linux, macOS, Windows, and Android. At the mo
...
+It's available on most package managers.
 
-At a minimum, you should run an NTP client on the server.
+Once installed, the default configuration is enough.
 
-> **Note:** Most operating systems (including macOS') automatically sync with NTP by default.
+At a minimum, you should run an NTP client (such as chrony) on the server.
 
-If the above sounds like latin to you, don't worry. You should be fine as long as you haven't messed around with the time and date settings on your computer (they should be set automatically).
+> **Note:** Most operating systems (including macOS) automatically sync with NTP by default, however it's still recommended to run chrony.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/install.md]: It's not recommened to run two NTP clients - they will fight each other

File: docs/the_nimbus_book/src/preparation.md [hunk 2]
```diff
@@ -5,26 +5,23 @@
 
 Please check that you are running the latest stable [Nimbus software release](https://github.com/status-im/nimbus-eth2/releases).
 
-> **Note:** If you are setting up your client before launch, it is your responsibility  to check for any new software releases in the run up to launch. At the minimum you should check the [release page](https://github.com/status-im/nimbus-eth2/releases) weekly.
+> At the minimum you should check the [release page](https://github.com/status-im/nimbus-eth2/releases) every other week.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/preparation.md]: Let's change this to a recommendation to subscribe to release notifications instead (via github releases for example)

File: docs/the_nimbus_book/src/start-syncing.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ To minimize the amount of downtime, you should ensure that your beacon node is [
 
 This is particularly important if you are joining a network that's been running for a while since the sync could take some time.
 
-> **N.B.** In order to process incoming validator deposits from the eth1 chain, you'll need to run an eth1 client (**web3 provider**) in parallel to your eth2 client. See [here](./eth1.md) for instructions on how to do so.
+> ⚠️ If you want to sync faster and attest immediately, you should take a look at [checkpoint sync](./trusted-node-sync.md)
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/start-syncing.md]:
```suggestion
> ⚠️ If you want to sync faster and attest immediately, you should take a look at [trusted node sync](./trusted-node-sync.md)
```

File: docs/the_nimbus_book/src/install.md (hunk 1 above)
Comment (TennisBowling) [reply docs/the_nimbus_book/src/install.md]: Right. It's probably favorable to use the built in on mac.

File: docs/the_nimbus_book/src/preparation.md (hunk 2 above)
Comment (TennisBowling) [reply docs/the_nimbus_book/src/preparation.md]: You can subscribe to github releases? Didn't know.
what about joining nimbus' discord?

Comment (etan-status) [reply docs/the_nimbus_book/src/preparation.md]: In top right corner of the repo, go to "Watch", then Custom > Releases.
The Beaconcha.in app also supports release notifications if you run a Nimbus validator.

Comment (arnetheduck) [reply docs/the_nimbus_book/src/preparation.md]: discord is fine as well - but if we have to pick one option, the release notifications are the lowest-noise option (along with our mailing list: https://subscribe.nimbus.team/ - actually - let's put the mailing list here)

File: docs/the_nimbus_book/src/hardware.md [hunk 4]
```diff
@@ -14,7 +14,7 @@
 
 In order to process incoming validator deposits from the eth1 chain, you will need to run an eth1 client in parallel to your eth2 client. While it is possible to use a third-party service like [Infura](/infura-guide.md), if you choose to run your own eth1 client locally, you'll need more memory and storage.
 
-For example, you'll need at least another **500GB SSD** to run [geth fast sync](/eth1.md) on mainnet.
+For example, you'll need at least another **1T SSD** to run [geth fast sync](/eth1.md) on mainnet.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/hardware.md]:
```suggestion
For example, you'll need at least another **1TB SSD** to run [geth fast sync](/eth1.md) on mainnet.
```

File: docs/the_nimbus_book/src/install.md [hunk 5]
```diff
@@ -6,17 +6,30 @@ The Nimbus beacon chain can run on Linux, macOS, Windows, and Android. At the mo
...
+# Archlinux, using an AUR manager
+yourAURmanager chrony
+```
+
+Chrony will uninstall any existing NTP servers.
+
+It's available on most package managers.
 
-At a minimum, you should run an NTP client on the server.
+Once installed, the default configuration is enough.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/install.md]:
```suggestion
Once installed, the default configuration is good enough.
```

File: docs/the_nimbus_book/src/install.md [hunk 6]
```diff
@@ -6,17 +6,30 @@ The Nimbus beacon chain can run on Linux, macOS, Windows, and Android. At the mo
...
 
-> **Note:** Most operating systems (including macOS') automatically sync with NTP by default.
+At a minimum, you should run an NTP client (such as chrony) on the server.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/install.md]:
```suggestion
At a minimum, you should run an NTP client (such as chrony) on the server.  Note that most operating systems (including macOS') automatically sync with NTP by default.
```

File: docs/the_nimbus_book/src/install.md [hunk 7]
```diff
@@ -6,17 +6,30 @@ The Nimbus beacon chain can run on Linux, macOS, Windows, and Android. At the mo
...
 
-If the above sounds like latin to you, don't worry. You should be fine as long as you haven't messed around with the time and date settings on your computer (they should be set automatically).
+> **Note:** If you're on MacOS, it's better to just use the built-in ntp server
```
Comment (unixpi) [inline docs/the_nimbus_book/src/install.md]:
```suggestion
If the above sounds like latin to you, don't worry. You should be fine as long as you haven't messed around with the time and date settings on your computer (they should be set automatically).
```

File: docs/the_nimbus_book/src/preparation.md (hunk 2 above)
Comment (unixpi) [inline docs/the_nimbus_book/src/preparation.md]:
```suggestion
> In order to stay on top of new releases you should subscribe to [our mailing list](https://subscribe.nimbus.team/).
```

File: docs/the_nimbus_book/src/preparation.md [hunk 8]
```diff
@@ -5,26 +5,23 @@
...
 
 ## More than 15 peers
 
-Please check that your node has at least 15 peers. To monitor your peer count, pay attention to the [`Slot start` messages in your logs](keep-an-eye.md#keep-track-of-your-syncing-progress).
+Please check that your node has at least 15 peers. To monitor your peer count, pay attention to the [`Slot start` messages in your logs](keep-an-eye.md#keep-track-of-your-syncing-progress). You should follow the [networking page](networking.md) to get good peers.
```
Comment (unixpi) [inline docs/the_nimbus_book/src/preparation.md]:
```suggestion
Please check that your node has at least 15 peers. To monitor your peer count, pay attention to the [`Slot start` messages in your logs](keep-an-eye.md#keep-track-of-your-syncing-progress). See the [networking page](networking.md) for more tips.
```

File: docs/the_nimbus_book/src/start-syncing.md [hunk 9]
```diff
@@ -4,9 +4,9 @@ To minimize the amount of downtime, you should ensure that your beacon node is [
 
 This is particularly important if you are joining a network that's been running for a while since the sync could take some time.
 
-> **N.B.** In order to process incoming validator deposits from the eth1 chain, you'll need to run an eth1 client (**web3 provider**) in parallel to your eth2 client. See [here](./eth1.md) for instructions on how to do so.
+> ⚠️ If you want to sync faster and attest immediately, you should take a look at [trusted node sync](./trusted-node-sync.md)
```
Comment (unixpi) [inline docs/the_nimbus_book/src/start-syncing.md]:
```suggestion
> **Tip:** If you'd like to sync faster and start attesting immediately, we recommend taking a look at [trusted node sync](./trusted-node-sync.md)
```

**Review by unixpi — COMMENTED** [review]
good work. left a few suggestions

**Comment by arnetheduck** [comment]
Thanks!

**Pull request merged**